publish-nokendra.sh

```diff
@@ -72,6 +72,10 @@ npm install
 npm run build
 popd
 
+pushd pca-ui/src/witch
+npm run build
+popd
+
 
 echo "Downloading witch file and upload into artifacts bucket"
```

Comment:
Should also remove the code that downloads the witch file, adds to artoifacts bucket, and the 'sed' code that inline edits WITCHKEY token